### DISCUSSION TITLE
Add to Administration the ability to change the timeout time when reporting to Kibana

### DISCUSSION THREAD

**Discussion by BigMacCat** [Security-Onion-Solutions/securityonion#13621] [Ideas]
Hi all!

When generating a CSV report in Kibana, the default timeout time is 240 seconds. This may not be enough if the report is too large. I propose to add the ability to change the timeout time for this feature (xpack.reporting.queue.timeout) through Administration.

**Comment by dougburks** [comment]
Please try the following:
- Go to Administration, click the Options dropdown, and then enable the `Show advanced settings` toggle.
- Navigate to kibana > advanced.
- Add the following config:
```
kibana:
  config:
    xpack:
      reporting:
        queue:
          timeout: 480
```
- Click the checkmark to save.
- Click the `SYNCHRONIZE GRID` button.

![image](https://github.com/user-attachments/assets/3be9b912-9309-4027-b24f-dc3eed5b422d)
